### PR TITLE
AB#824 Create search box 

### PR DESCRIPTION
**Pull request by szymonbla**

### CODE REVIEW

**Comment by JKrymarys** [comment]
The paddings between placeholder/searched text seems to be too much... definetely the components looks a bit bigs comparing to mocks. 

In figma there is 16px, while you have 4px (on text) and 16px (between text and border) 

image

image

**Comment by JKrymarys** [comment]
Looking deeper, there is also some padding on button/icon element. Anyway instead of overriding everything I'd change padding `<Box>` (Seachbox.tsx line 11) that all combined gives desired value (and look good :)).

File: src/common/components/SearchBox/SearchBox.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  return (
+    <Box
+      sx={{
+        width: '100%',
+        display: 'flex',
+        justifyContent: 'space-between',
+        border: '1.5px solid',
+        borderColor: 'grey.400',
+        borderRadius: 2,
+        p: (theme) => theme.spacing(2)
```
Comment (JKrymarys) [inline src/common/components/SearchBox/SearchBox.tsx:19]: Adjust the padding to match designs (I explained it more in a PR comment)

**Review by JKrymarys — CHANGES_REQUESTED** [review]

File: src/common/components/SearchBox/SearchBox.tsx (hunk 1 above)
Comment (szymonbla) [reply src/common/components/SearchBox/SearchBox.tsx]: Thanks for the suggestions. Summing up, I moved padding to the ```MuiInputBase-input```. It seems that's work. Additionally, I adjusted the button/icon search styling, I hope now it's better

**Review by JKrymarys — APPROVED** [review]
LGTM now

**Pull request merged**